lmdeploy/serve/async_engine.py

```diff
@@ -353,6 +353,7 @@ async def get_generator(self, stop: bool, session_id: int):
     def batch_infer(
             self,
             prompts: Union[List[str], str, List[Dict], List[List[Dict]]],
+            session_ids: Union[List[int], int] = None,
```

Comment:
Introducing `session_ids` will make the API hard to understand.
Is there any way we can handle it internally?

Reply:
The stream_infer supports batch inputs, we need to match the inputs and outputs and we distinguish it by session_id currently. If we don't introduce the session_ids, we need to change the [output](https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/serve/async_engine.py#L501-L504) like below which i present the ith input in the batch.
```
                i, out = outputs.get(timeout=0.001)
                if out is None:
                    break
                yield i, out
```